src/app/public/modules/ag-grid/ag-grid-data-manager-adapter.directive.ts

```diff
@@ -171,7 +172,7 @@ export class SkyAgGridDataManagerAdapterDirective implements AfterContentInit, O
       .pipe(takeUntil(this.ngUnsubscribe))
       .subscribe((event: ColumnMovedEvent) => {
         let columnOrder = agGrid.columnApi.getAllDisplayedVirtualColumns().map(
-          col => col.getColDef().colId
+          col => col.getColDef().colId || col.getColDef().field
```

Comment:
Ugh..... this gives me heartburn..... but I don't know of a better way to do this......